ee/tables/dataflattentable/exec_and_parse.go

```diff
@@ -48,12 +49,34 @@ func WithTableDebug() execTableV2Opt {
 	}
 }
 
+// WithIncludeStderr combines stdout and stderr before attempting any parsing
 func WithIncludeStderr() execTableV2Opt {
 	return func(t *execTableV2) {
 		t.includeStderr = true
 	}
 }
 
+// WithReportStderr will include stderr (if populated) in the parsed output as a
+// separate row in the results produced by the query
+func WithReportStderr() execTableV2Opt {
+	return func(t *execTableV2) {
+		t.reportStderr = true
+	}
+}
+
+// WithReportMissingBinary will include an error row in the results
+// indicating that the binary is missing. Without this option, queries
+// against missing binaries typically return no results, with the error
+// being ignored.
+// Note that for tables that run through macos RunDisclaimed, we cannot pass our missing
+// binary errors back through- this information is conveyed through stderr, so callers
+// should also include the WithReportStderr option to see the same behavior there.
+func WithReportMissingBinary() execTableV2Opt {
```

Comment:
Do you think we'll ever want not use `WithReportMissingBinary` and `WithReportStderr` together?

Reply:
I was surprised we didn't have a need for this before, but don't understand the upstream impacts enough to make a call on default behavior. There seems to be a pattern of no binary == no data everywhere else, but I wasn't sure what (if anything) depends on that assumption from checks.

Either way i'm fine with making this one option if that feels cleaner, our only documented use case for this (zscaler) does want both